### ISSUE TITLE
[BUG] Videos not playing

### ISSUE THREAD

**Issue by SherlockJomesBG** [IacobIonut01/Gallery#542]
## [Delete before submission] Please use and reproduce the bug on the latest build from Github Actions (Artifacts)
**Describe the bug**

**To Reproduce**
1. Go to Gallery App
2. Click on Video

**Expected behavior**
The video to start playing automatically or to start playing when the play button was clicked

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Xiaomi Poco X4 Pro 5G
 - OS: Android 14 (Evolution X) (Custom ROM)

**Additional context**
Videos are correctly found, when clicked it displays the runtime and all metadata associated with it! It just never actually plays the video file!

**Comment by IacobIonut01** [comment]
There's no error on playing videos, please include a screen record, app version...

**Comment by SherlockJomesBG** [comment]
> There's no error on playing videos, please include a screen record, app version...

https://github.com/user-attachments/assets/72218a66-9082-4c77-9a56-60c2e7d010fd

V3.0.1

**Comment by IacobIonut01** [comment]
Include a logcat, make sure videos work on other apps

**Comment by SherlockJomesBG** [comment]
> Include a logcat, make sure videos work on other apps

Ignore my stupidity here! How do I obtain a logcat?

**Comment by SherlockJomesBG** [comment]
> Include a logcat, make sure videos work on other apps

To update you! Some videos do play and some do not! They are all .MP4 files! The videos that don't play do play in other apps

**Comment by WhitePeace36** [comment]
same issue here for me, but the strange thing is some .mp4 files work and others don't. 

And when i open a video in another app and it asks me where i want to open it and i say Gallery then it plays fine. But when i open the same video in Gallery directly it doesn't work.

**Comment by M0-7** [comment]
I have the same issue, although you can still play the video by clicking on a random timestamp

**Comment by IacobIonut01** [comment]
> I have the same issue, although you can still play the video by clicking on a random timestamp

Check the latest nightly

**Comment by M0-7** [comment]
Yep, this release seems to solve the problem. Also is this the release in which the app is going to be published on Accrescent?

**Comment by IacobIonut01** [comment]
Accrescent is currently invitation based only for developers and I don't have one, which means I can't upload it

**Comment by M0-7** [comment]
That sucks. Oh well it's fine it's eventually going to be opened up to all developers

**Comment by muelsyse-az** [comment]
Is this issue still occuring in the latest version (v3.1.2)?

I can play videos just fine on the latest nightly at least. Otherwise, this can be closed.
